pkg/acceptance/helpers/secret_client.go

```diff
@@ -71,7 +71,7 @@ func (c *SecretClient) CreateWithBasicAuthenticationFlow(t *testing.T, id sdk.Sc
 func (c *SecretClient) CreateWithGenericString(t *testing.T, id sdk.SchemaObjectIdentifier, secretString string) (*sdk.Secret, func()) {
 	t.Helper()
 	ctx := context.Background()
-	request := sdk.NewCreateWithGenericStringSecretRequest(id, secretString)
+	request := sdk.NewCreateWithGenericStringSecretRequest(id, secretString).WithOrReplace(true)
```

Comment:
We should not use OrReplace by default. It should be either an argument, or the function should be called `CreateOrReplace...` or `Recreate...`

Reply:
makes sense, the `WithReplace` was not necessary so it will be removed